tools/gen_addon_readme.py

```diff
@@ -150,17 +179,32 @@ def gen_one_addon_readme(repo_name, branch, addon_name, addon_dir, manifest):
             repo_name=repo_name,
             runbot_id=runbot_id,
         ))
-    with open(os.devnull, 'w') as devnull:
-        r = subprocess.call([
-            'rst2html4.py', readme_filename,
-        ], stdout=devnull)
-        if r != 0:
-            raise click.ClickException(
-                "Error validating %s" % (readme_filename, )
-            )
     return readme_filename
 
 
+def gen_one_addon_index(readme_filename):
+    addon_dir = os.path.dirname(readme_filename)
+    index_dir = os.path.join(addon_dir, 'static', 'description')
+    index_filename = os.path.join(index_dir, 'index.html')
+    if os.path.exists(index_filename):
+        with open(index_filename) as f:
+            if 'oca-gen-addon-readme' not in f.read():
+                # index was created manually
+                return
+    try:
+        if not os.path.isdir(index_dir):
+            os.makedirs(index_dir)
+        publish_file(
+            source_path=readme_filename,
+            destination_path=index_filename,
+            writer_name='html4css1',
+            settings_overrides=RST2HTML_SETTINGS,
+        )
+        return index_filename
+    except ApplicationError:
+        raise click.ClickException("Error validating README.rst")
```

Comment:
what about a message with the `readme_filename` variable? ("Error validating %s" or similar)